hudi-bench/src/main/java/org/apache/hudi/bench/DeltaOutputType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.bench;
+
+/**
+ * Supported sink types
+ */
+public enum DeltaOutputType {
```

Comment:
rename: SinkType? Also this is more like InputSource right.. not sink ? again not very clear to me. IIRC I raised the same concerns on the old pr

Reply:
Renamed it, I did incorporate most of the feedback from the old PR, some were missed which I've tried to rename, please see if it makes sense to you